explore/interface.py

```diff
@@ -93,8 +93,12 @@ def default(cls, value, **kw):
                 return cls(value)
 
 
-sech = lambda x: 1 / cosh(x)
-asech = lambda x: acosh(1 / x)
+def sech(x):
```

Comment:
this change results in a different object for `sech` and `asech` - does it affect their intended use?

Reply:
Ah, ok thanks.  In some places in bumps we do very subtle things where the difference between a lambda and a function matters a lot, but not here.